src/Interpreters/OpenTelemetryLog.cpp

```diff
@@ -0,0 +1,46 @@
+#include "OpenTelemetryLog.h"
+
+#include <DataTypes/DataTypeArray.h>
+#include <DataTypes/DataTypeDate.h>
+#include <DataTypes/DataTypeDateTime.h>
+#include <DataTypes/DataTypesNumber.h>
+#include <DataTypes/DataTypeString.h>
+#include <DataTypes/DataTypeUUID.h>
+
+namespace DB
+{
+
+Block OpenTelemetrySpanLogElement::createBlock()
+{
+    return {
+        {std::make_shared<DataTypeUUID>(), "trace_id"},
```

Comment:
Trace ID seems to be a 128 bit integer (32 char hex [according to spec](https://www.w3.org/TR/trace-context/#versioning-of-traceparent)).

UUIDv4 is a 128 bit integer and it can be converted to a hex via `hex(UUIDStringToNum(toString(trace_id)))`, but I'm not sure if it's the right logical format to store the value here.

Reply:
We got big int types recently, but they are still very experimental, so I'll keep the UUID for now.